kernel/src/schema/mod.rs

```diff
@@ -141,6 +142,7 @@ impl MetadataColumnSpec {
             Self::RowIndex => "row_index",
             Self::RowId => "row_id",
             Self::RowCommitVersion => "row_commit_version",
+            Self::FileName => "_file",
```

Comment:
at some point we should probably agree on what we want to call these :)

Reply:
Naming was taken from iceberg spec, but I agree FilePath for the variable name is better lets do that.